packages/terra-popup/src/PopupArrow.module.scss

```diff
@@ -3,82 +3,86 @@
       top: 50%;
       width: 0;
-      z-index: var(--terra-popup-arrow-z-index, 7002);
+      z-index: 7002;
 
       &::after {
-        border: var(--terra-popup-arrow-inner-size, 10px) solid transparent;
+        border-color: transparent;
+        border-style: solid;
+        border-width: var(--terra-popup-arrow-after-border-width, 10px);
         content: '';
         height: 0;
         position: absolute;
         width: 0;
-        z-index: var(--terra-popup-arrow-inner-z-index, 7003);
+        z-index: 7003;
       }
     }
 
     .popup-arrow[data-align-arrow='top'] {
-      border-bottom-color: var(--terra-popup-arrow-color, #b2b5b6);
+      border-bottom-color: var(--terra-popup-arrow-border-bottom-color, #b2b5b6);
       border-top-width: 0;
       display: block;
       top: 0;
       transform: translate3d(-100%, -100%, 0);
 
       &::after {
-        border-bottom-color: var(--terra-popup-arrow-inner-color, #fff);
+        border-bottom-color: var(--terra-popup-arrow-after-border-bottom-color, #fff);
         border-top-width: 0;
-        left: calc(-1 * var(--terra-popup-arrow-inner-size, 10px));
-        top: var(--terra-popup-arrow-inner-inset-size, 2px);
+        left: calc(-1 * var(--terra-popup-align-arrow-top-after-left, 10px));
+        top: var(--terra-popup-align-arrow-top-after-top, 2px);
       }
     }
 
     .popup-arrow[data-align-arrow='bottom'] {
       border-bottom-width: 0;
-      border-top-color: var(--terra-popup-arrow-color, #b2b5b6);
+      border-top-color: var(--terra-popup-arrow-border-top-color, #b2b5b6);
       display: block;
       top: 100%;
       transform: translate3d(-100%, 0, 0);
 
       &::after {
         border-bottom-width: 0;
-        border-top-color: var(--terra-popup-arrow-inner-color, #fff);
-        bottom: var(--terra-popup-arrow-inner-inset-size, 2px);
-        left: calc(-1 * var(--terra-popup-arrow-inner-size, 10px));
+        border-top-color: var(--terra-popup-arrow-after-border-top-color, #fff);
+        bottom: var(--terra-popup-align-arrow-bottom-after-bottom, 2px);
+        left: calc(-1 * var(--terra-popup-align-arrow-bottom-after-left, 10px));
       }
     }
 
     .popup-arrow[data-align-arrow='left'] {
       border-left-width: 0;
-      border-right-color: var(--terra-popup-arrow-color, #b2b5b6);
+      border-right-color: var(--terra-popup-arrow-border-right-color, #b2b5b6);
       display: block;
       left: 0;
       transform: translate3d(-100%, -100%, 0);
 
       &::after {
         border-left-width: 0;
-        border-right-color: var(--terra-popup-arrow-inner-color, #fff);
-        left: var(--terra-popup-arrow-inner-inset-size, 2px);
-        top: calc(-1 * var(--terra-popup-arrow-inner-size, 10px));
+        border-right-color: var(--terra-popup-arrow-after-border-right-color, #fff);
+        left: var(--terra-popup-align-arrow-left-after-left, 2px);
+        top: calc(-1 * var(--terra-popup-align-arrow-left-after-top, 10px));
```

Comment:
I'm having a hard time figuring out the intent of these. Is the additional left intentional in both of these? `left-after-left` and `left-after-top`. 

border-right-color doesn't have an `align` prefix, but the other two do.

Reply:
Is `left` the arrow direction from the data selector above?

Reply:
So the first `left` in `left-after-left` is supposed to be arrow direction. I believe they were probably separated like this becasue the left, top, right, and bottom directions needed a secondary qualifer for them but the `border-right-color` property did not since those were unique to each arrow direction.

For clarity, we should probably maintain a unique identifier for each of them in the form of arrow direction, though it might be better to swap it to something like `--terra-popup-left-align-arrow-after-left`

Reply:
Yeah, `--terra-popup-left-align-arrow-after-left` reads better in mind, easier to reason about what it is for.

Reply:
Fixed here https://github.com/cerner/terra-framework/pull/439/commits/2621fc4230b26560a600e1cc1474eb6c901de673